atkinson/logging/logger.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python
+"""Interface to get a configured logger"""
+
+import importlib
+
+
+def getLogger(config=None):
+    """
+    Returns a logger object of the type specified.
+
+    :param config: Dictionary of values to configure the desired logger object.
+
+    For example, to use the default python logger, you can
+    either simply pass None, or specify any details supported by
+    the atkinson default driver for the python logger.  You
+    would specify the default logger (with custom loglevel) with:
+    ::
+
+      {'driver':'atkinson.logging.drivers.default',
+      'root':{'level':DEBUG}}
+
+    The atkinson logging wrapper code specifies a toplevel element in the config
+    dict of 'driver', pointing at the driver that will load the desired
+    configuration and return a logger to use
+
+    """
+    if not config:
+        config = {'driver': 'atkinson.logging.drivers.default'}
+    module = importlib.import_module(config.get('driver'))
```

Comment:
Might want a try..except here just in case the import fails. Then either raise an exception and/or use the default logger to show that a bad module was given.

Reply:
Good idea, I'll add something along those lines!